### PR TITLE
BUG: numpy.ma.round works on zero dimensional arrays

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: numpy/ma/core.py [hunk 1]
```diff
@@ -5107,8 +5107,12 @@ def round(self, decimals=0, out=None):
 
         """
         result = self._data.round(decimals=decimals, out=out).view(type(self))
-        result._mask = self._mask
-        result._update_from(self)
+        if result.ndim:
```
Comment (charris) [inline numpy/ma/core.py]: It would be more informative to write `if result.ndim > 0:`, as that exactly expresses the desired condition.

**Comment by charris** [comment]
LGTM modulo minor nitpick.

**Comment by jjhelmus** [comment]
Updated with suggested if statement.

**Comment by charris** [comment]
Thanks @jjhelmus .

**Pull request merged**